gradle/libs.versions.toml

```diff
@@ -61,7 +61,7 @@ reflections = "0.10.2"
 retrofit = "3.0.0"
 s3 = "2.31.48"
 saxonHe = "12.7"
-scanoss = "0.10.1"
+scanoss = "0.11.0"
```

Comment:
The commit message title should start with `build(gradle):` as the library is made globally available in Gradle, and not just to the scanoss module.